providers/base/tests/test_networking_http.py

```diff
@@ -28,33 +28,24 @@
 
 class NetworkingHTTPTests(TestCase):
     @patch("networking_http.subprocess.run")
-    @patch("networking_http.time.sleep")
-    def test_http_connect_max_retries(self, mock_sleep, mock_run):
-        with self.assertRaises(SystemExit):
-            networking_http.http_connect("test", 0)
-
-    @patch("networking_http.subprocess.run")
-    @patch("networking_http.time.sleep")
-    def test_http_connect_success(self, mock_sleep, mock_run):
+    def test_http_connect_success(self, mock_run):
         """
         Test that `http_connect` returns safely if the wget command returns 0
         """
-        self.assertEqual(networking_http.http_connect("test", 3), None)
+        self.assertEqual(networking_http.http_connect("test"), None)
 
     @patch("networking_http.subprocess.run")
-    @patch("networking_http.time.sleep")
+    @patch("checkbox_support.helpers.retry.time.sleep")
```

Comment:
If you mocked time sleep like I suggested above, this wouldn't need to be changed!

Reply:
Fixed in https://github.com/canonical/checkbox/commit/ae74ca793f77e155aa2b6e2b7b6b836966176e2a